specs/derivation.md

```diff
@@ -945,20 +934,20 @@ epoch sequencing window, assuming it is in the same epoch as `safeL2Head`).
 > could post a batch for the L2 block `safeL2Head + 1` on L1 block `safeL2Head.1Origin`.
 
 Keeping things worst case, `safeL2Head.l1Origin` would also be the last allowable block for the frame to land. The
-allowed time range for frames within a channel to land on L1 is `[channel_id.timestamp, channel_id.timestamp +
-CHANNEL_TIMEOUT]`. The allowed L1 block range for these frames are any L1 block whose timestamp falls inside this time
+allowed time range for frames within a channel to land on L1 is `[channel_id.number, channel_id.number +
+CHANNEL_TIMEOUT]`. The allowed L1 block range for these frames are any L1 block whose number falls inside this block
 range.
 
-Therefore, to be safe, we can reset the L1 head of Channel Buffering to the oldest L1 block whose timestamp is higher
-than `safeL2Head.l1Origin.timestamp - CHANNEL_TIMEOUT`.
+Therefore, to be safe, we can reset the L1 head of Channel Buffering to the L1 block whose number is
+`safeL2Head.l1Origin.number - CHANNEL_TIMEOUT`.
 
 > **Note** The above is what the implementation currently does.
 
 In reality it's only strictly necessary to reset the oldest L1 block whose timestamp is higher than the oldest
 `channel_id.timestamp` found in the batcher transaction that is not older than `safeL2Head.l1Origin.timestamp -
 CHANNEL_TIMEOUT`.
 
-We define `CHANNEL_TIMEOUT = 600`, i.e. 10 hours.
+We define `CHANNEL_TIMEOUT = 600`, i.e. 10 hours. (600s == 10 mins. Choose order 500 blocks as channel timeout?)
```

Comment:
`CHANNEL_TIMEOUT * L1_BLOCK_TIME = 600 * 12 seconds = 2 hours`.
We should set it to `10*60/12=50`

And update the configuration in op-e2e and chain-ops as well (divide by 12).